src/main/java/org/eln2/mc/common/blocks/cell/VoltageSourceCellBlock.kt

```diff
@@ -11,14 +12,19 @@ import net.minecraft.world.MenuProvider
 import net.minecraft.world.entity.player.Inventory
 import net.minecraft.world.entity.player.Player
 import net.minecraft.world.inventory.AbstractContainerMenu
+import net.minecraft.world.level.BlockGetter
 import net.minecraft.world.level.Level
+import net.minecraft.world.level.block.Block
 import net.minecraft.world.level.block.state.BlockState
 import net.minecraft.world.phys.BlockHitResult
+import net.minecraft.world.phys.shapes.CollisionContext
+import net.minecraft.world.phys.shapes.VoxelShape
 import net.minecraftforge.network.NetworkHooks
 import org.eln2.mc.common.blocks.CellBlockBase
 import org.eln2.mc.common.blocks.CellTileEntity
 import org.eln2.mc.common.cell.CellRegistry
 import org.eln2.mc.common.containers.VoltageSourceCellContainer
+import java.util.function.Function
```

Comment:
These added imports can probably be removed.

Reply:
Removed the unnecessary imports.